### PR TITLE
Support Solana

### PR DESCRIPTION
**Pull request by vickygos**
- Allows Solana to be scored
- Adds test to verify
- Minor version upgrade to 0.6.0

### CODE REVIEW

File: open_rarity/scoring/scorer.py [hunk 1]
```diff
@@ -33,9 +33,14 @@ def validate_collection(self, collection: Collection) -> None:
                 "numeric or date traits"
             )
 
-        if collection.token_standards != [TokenStandard.ERC721]:
+        allowed_standards = {
+            TokenStandard.ERC721,
+            TokenStandard.METAPLEX_NON_FUNGIBLE,
+        }
```
Comment (vickygos) [inline open_rarity/scoring/scorer.py:39]: FYI: Thought about putting it in a constant or env var but felt it wasn't worth given we are addressing 1155 soon

Comment (vickygos) [reply open_rarity/scoring/scorer.py:39]: Also didn't want people to do funky stuff like forcibly overwriting them :)

**Comment by grantli-os** [comment]
Should we add a `Token.from_metaplex` helper function (similar to how we have `Token.from_erc721`)?

**Review by grantli-os — APPROVED** [review]

**Review by theelderbeever — APPROVED** [review]
LGTM

**Pull request merged**